Triphouse-project/index.html

```diff
@@ -8,7 +8,61 @@
     <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-    <main class="offer">
+    <header class="header">
+        <div class="header__wrapper">
+             <div class="header__logo">
+                 <a href="#" class="heder__logo-link">
+                     <img src="./ikons/logo_vector.svg" alt="TripHouse" class="heder_logo-img">
+                 </a>
+             </div>
+             <nav class="header__nav">
+                 <ul class="heder__list">
+                     <li class="header__item">
+                         <a href="#" class="heder__link">Stays</a>
+                     </li>
+                     <li class="header__item">
+                         <a href="#" class="heder__link">Attractions</a>
+                     </li>
+                     <li class="header__item">
+                        <a href="#" class="header__link">
+                          <img src="./ikons/Night.svg" alt="night" />
+                        </a>
+                      </li>
+                      <li class="header__item">
+                        <a href="#" class="header__link">
+                          <img src="./ikons/AccountCircle.svg" alt="AcccountCircle" />
+                        </a>
+                      </li>
+                 </ul>
+             </nav>    
+        </div>
+        <div class="content">
+            <div class="content__title">
+              <p class="content-title__text">Discover stays to live, work or just relax</p>
+              <div class="content__sbar">
+                <ul class="sbar__lables">
+                  <li class="sbar-lables__destination">Your destination or hotel name</li>
+                  <li class="sbar-lables__date">Check-in — Check-out</li>
+                </ul>
+              </div>
+              <div class="sbar__input">
+                <input type="text" class="sbar-input__destination" />
+                <input type="date" name="check-in" id="check-in" class="sbar__input" />
+                <input type="date" name="check-out" id="check-out" class="sbar__input" />
+                <input type="text" class="sbar__input" />
+                <input type="button" value="search" class="sbar__button" />
+              </div>
+              <div class="market__links">
+                <p>
+                  <img src="./img/google-play-badge.svg" alt="google play market" />
+                </p>
+                <p>
+                  <img src="./img/Download_on_the_App_Store_Badge_US-UK_RGB_blk_092917 1.svg" alt="appStore" />
+                </p>
+              </div>
+            </div>
+    </header>
+    <!-- <main class="offer">
```

Comment:
Why is it commented?

Reply:
because when I worked I commented out, to make it more convenient, I forgot back.